strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-raw-layout-provider/src/test/java/org/carlspring/strongbox/providers/layout/RawLayoutProviderTest.java

```diff
@@ -68,69 +60,34 @@
     @Inject
     private RepositoryPathResolver repositoryPathResolver;
 
-
-    @BeforeAll
-    public static void cleanUp()
-            throws Exception
-    {
-        cleanUp(getRepositoriesToClean());
-    }
-
-    public static Set<RepositoryDto> getRepositoriesToClean()
-    {
-        Set<RepositoryDto> repositories = new LinkedHashSet<>();
-        repositories.add(createRepositoryMock(STORAGE, REPOSITORY, RawLayoutProvider.ALIAS));
-
-        return repositories;
-    }
-
-    @BeforeEach
-    public void setUp()
-            throws Exception
-    {
-        Configuration configuration = configurationManagementService.getConfiguration();
-
-        if (configuration.getStorage(STORAGE) == null)
-        {
-            createStorage(STORAGE);
-        }
-
-        if (configuration.getStorage(STORAGE).getRepository(REPOSITORY) == null)
-        {
-            createRepository(STORAGE, REPOSITORY);
-        }
-    }
-
-    @AfterEach
-    public void removeRepositories()
-            throws IOException, JAXBException
-    {
-        removeRepositories(getRepositoriesToClean());
-    }
-
     @Test
-    public void testNullArtifactCoordinates()
+    void testNullArtifactCoordinates()
     {
         NullArtifactCoordinates coordinates = new NullArtifactCoordinates("foo/bar/blah.bz2");
 
-        System.out.println("coordinates.toPath(): " + coordinates.toPath());
+        logger.info("coordinates.toPath(): {}", coordinates.toPath());
     }
 
+    @ExtendWith({RepositoryManagementTestExecutionListener.class,
+                 ArtifactManagementTestExecutionListener.class })
     @Test
-    public void testDeployAndResolveArtifact()
+    void testDeployAndResolveArtifact(@TestRepository(layout = RawLayoutProvider.ALIAS,
+                                                      storageId = STORAGE,
+                                                      repositoryId = REPOSITORY)
+                                     Repository repository,
+                                     @TestArtifact(storageId = STORAGE,
```

Comment:
@ptirador we should deploy artifact within test case, so can you please remove `storageId` and `repositoryId` from this annotation.